src/utils/expressionToFunction.ts

```diff
@@ -0,0 +1,8 @@
+import { Unit } from './../generated/sdk';
+const expressionToFunction = (expression: string) =>
+  new Function('x', `return ${expression}`);
```

Comment:
Are there any security risks with allowing user defined code here?

Reply:
Good point @TickleThePanda , I talked with the team and we decided to put sanitization before data is saved. https://github.com/UserOfficeProject/user-office-backend/pull/557/commits/3b41c148fe0e0998393107a867bee4db0ea20286